### ISSUE TITLE
[TF2] "Spectator Target ID Location" settings other than default are misplaced.

### ISSUE THREAD

**Issue by NonhumanFromSteam** [ValveSoftware/Source-1-Games#3976]
The first time the spectator targetID appears it is placed incorrectly if a spectator targetid location setting other than default is selected 
( on the first spectated player after your death, it appears misplaced. ):
![image](https://user-images.githubusercontent.com/25296372/175952944-5d9a95b2-da6a-4099-8303-cf0c6f8f039b.png)

But after switching the spectator target it gets fixed though only for one instance:
![image](https://user-images.githubusercontent.com/25296372/175953299-aed9d9f5-2f49-4957-b232-b92cbe155529.png)

Basically after every death it will still be misplaced on the first spectated player until you switch to another spectated player.

This is the setting I use, but it also happens on bottom left or bottom right corner settings too:
![image](https://user-images.githubusercontent.com/25296372/175953789-73dc7e1c-e172-48bd-9df8-d9c653c1b24d.png)
